plugins/browser-plugin-media-tracking/src/types.ts

```diff
@@ -0,0 +1,55 @@
+import { MediaElement, MediaPlayer, MediaPlayerEvent, VideoElement } from './contexts';
+import { SnowplowMediaEvent } from './snowplowEvents';
+import { MediaEvent, TextTrackEvent, DocumentEvent, VideoEvent } from './mediaEvents';
+
+export type EventGroup = (DocumentEvent | MediaEvent | SnowplowMediaEvent | TextTrackEvent | VideoEvent | string)[];
+
+export type MediaEventType = DocumentEvent | MediaEvent | SnowplowMediaEvent | TextTrackEvent | VideoEvent;
+
+export type HTMLVideoFormat = 'mp4' | 'ogg' | 'webm';
+
+// All Video formats can be used as Audio as well
+export type HTMLAudioFormat = 'aac' | 'aacp' | 'caf' | 'flac' | 'mp3' | 'wav' | HTMLVideoFormat;
```

Comment:
Same comment here as I mentioned earlier. I wonder if we even really need these and they can just be strings.